content/kb/tutorials/databases/snowflake.md

```diff
@@ -58,18 +56,19 @@ Once you have executed the queries, you should see a preview of the table in the
 
 Make sure to note down the name of your warehouse, database, and schema. ☝️
 
-## Install snowflake-snowpark-python
+## Install snowflake-connector-python
 
-You can find the instructions and prerequisites for installing `snowflake-snowpark-python` in the [Snowpark Developer Guide](https://docs.snowflake.com/en/developer-guide/snowpark/python/setup).
+You can find the instructions and prerequisites for installing `snowflake-connector-python` in the [Snowflake Developer Guide](https://docs.snowflake.com/en/developer-guide/python-connector/python-connector-install).
 
 ```bash
-pip install "snowflake-snowpark-python[pandas]"
+pip install "snowflake-connector-python"
 ```
 
-Particular prerequisites to highlight:
+<Important>
+
+Ensure you have the correct pyarrow version installed for your version of `snowflake-connector-python`. When in doubt, try uninstalling `pyarrow` before installing `snowflake-connector-python`.
 
-- Currently, only python 3.8 is supported.
-- Ensure you have the correct pyarrow version installed for your version of `snowflake-snowpark-python`. When in doubt, try uninstalling pyarrow before installing snowflake-snowpark-python.
+</Important>
```

Comment:
I think this was just an issue for the snowpark driver, so it can be removed now that we just use the snowflake connector

Reply:
I was looking at the Snowflake documentation and I did find some [KB](https://community.snowflake.com/s/article/You-cannot-use-snowflake-connector-python-with-pyarrow-from-conda-forge-on-linux) articles about finicky Pyarrow versions with the snowflake-connector-python as well. So just want to double-check before I remove it.

Reply:
OK! It's a small thing so no need to block this PR on further research there. Fine by me to just leave it as is.